src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -38,25 +42,65 @@ export const ADMIN_COOKIE_OPTIONS_WITH_EXPIRY = {
 
 const logger = createLoggerWithLabel(module)
 
-const serveFormReact: ControllerHandler = (_req, res) => {
-  const reactFrontendPath = path.resolve('dist/frontend')
-  logger.info({
-    message: 'serveFormReact',
-    meta: {
-      action: 'routeReact.serveFormReact',
-      __dirname,
-      cwd: process.cwd(),
-      reactFrontendPath,
-    },
-  })
-  return (
-    res
-      // Prevent index.html from being cached by browsers.
-      .setHeader('Cache-Control', 'no-cache')
-      .sendFile(path.join(reactFrontendPath, 'index.html'))
-  )
+const reactFrontendPath = path.resolve('dist/frontend')
+const reactHtml = readFileSync(path.join(reactFrontendPath, 'index.html'), {
+  encoding: 'utf8',
+})
+
+type MetaTags = {
+  title: string
+  description: string
+  image: string
+}
+const replaceWithMetaTags = ({
+  title,
+  description,
+  image,
+}: MetaTags): string => {
+  return reactHtml
+    .replace(/(__OG_TITLE__)/g, DOMPurify.sanitize(title))
+    .replace(
+      /(<title>__OG_TITLE__<\/title>)/g,
+      DOMPurify.sanitize(`<title>${title}</title>`),
+    )
```

Comment:
Suggestion: in index.html, replace `<title>FormSG</title>` by `<title>__OG_TITLE__</title>`, and reduce the replacement code to this:

```suggestion
    .replace(/(__OG_TITLE__)/g, DOMPurify.sanitize(title))
```